packages/ui/src/modal/buy/BuyModalRenderer.tsx

```diff
@@ -78,6 +79,8 @@ type Props = {
   children: (props: ChildrenProps) => ReactNode
 }
 
+const CHAIN_ID = process.env.CHAIN_ID ? +process.env.CHAIN_ID : 1
```

Comment:
personal preference but i find
`Number(process.env.CHAIN_ID)` to be easier to read

Reply:
I'll update this.